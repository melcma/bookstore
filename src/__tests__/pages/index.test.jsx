import { render, screen } from "@testing-library/react";
import HomePage from "../../pages/index";
import "@testing-library/jest-dom";

describe("HomePage", () => {
  it("renders a heading", () => {
    render(<HomePage />);

    const heading = screen.getByRole("heading", {
      name: /Hello/i,
    });

    expect(heading).toBeInTheDocument();
  });

  it("renders homepage unchanged", () => {
    const { container } = render(<HomePage />);
    expect(container).toMatchSnapshot();
  });
});
