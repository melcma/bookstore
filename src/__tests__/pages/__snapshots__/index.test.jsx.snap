// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders homepage unchanged 1`] = `
<div>
  <div>
    <nav
      class="nav"
    >
      <a
        href="/"
      >
        Home
      </a>
      <a
        href="/products"
      >
        Shop
      </a>
      <a
        href="/cart"
      >
        Cart
      </a>
    </nav>
    <h1>
      Hello
    </h1>
  </div>
</div>
`;
